src/Bridge/Symfony/Bundle/DependencyInjection/ApiPlatformExtension.php

```diff
@@ -141,7 +141,13 @@ private function registerSwaggerConfiguration(ContainerBuilder $container, array
         }
 
         $loader->load('swagger.xml');
-        $container->setParameter('api_platform.enable_swagger', (string) $config['enable_swagger']);
+
+        if ($config['enable_swagger_ui']) {
+            $loader->load('swagger-ui.xml');
+            $container->setParameter('api_platform.enable_swagger_ui', $config['enable_swagger_ui']);
```

Comment:
OH: We don't net to set it for the container if it's false anyway. Like we have done with enable_swagger.